src/bitmessageqt/languagebox.py

```diff
@@ -8,24 +10,27 @@
 
 
 class LanguageBox(QtGui.QComboBox):
+    """LanguageBox class for QT UI"""
```

Comment:
Qt is not QT

Reply:
@g1itch sorry for that give me a minute,I am fixing it right now.

Reply:
fixed.